docs/data-model.rst

```diff
@@ -668,6 +668,9 @@ mutation does not result in any change of state. This error is
 raised at run-time when we reconstruct sample genotypes, for example
 in the :meth:`TreeSequence.variants` iterator.
 
+.. note:: As ``tskit.UNKNOWN_TIME`` is implemented as a ``NaN`` value, tests for
+    equality will always fail. Use ``util.is_unknown_time`` to detect unknown
```

Comment:
We don't expose the package structure - just ``tskit.is_unknown_time`` presumably?

Reply:
Opps, keep tripping over this one. Fixed.